src/main/java/sg/ncl/testbed_interface/SignUpMergedForm.java

```diff
@@ -38,36 +38,35 @@
     private String institution;
 
     @NotEmpty(message = "Institution Abbreviation cannot be empty")
-    private String institutionAbbreviation;
+    private String institutionAbbreviation = "defaultAbbrev";
 
     @NotEmpty(message = "Website cannot be empty")
-    private String website;
+    private String website = "http://default.com";
 
     @NotEmpty(message = "Address 1 cannot be empty")
-    private String address1;
+    private String address1 = "default";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "default" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)